### PR TITLE
chore: add skip_crds parameter to helm functions

### PR DESCRIPTION
**Pull request by kahirokunn**
https://github.com/tilt-dev/tilt/issues/6181

### CODE REVIEW

File: internal/tiltfile/files.go [hunk 1]
```diff
@@ -196,6 +196,7 @@ func (s *tiltfileState) helm(thread *starlark.Thread, fn *starlark.Builtin, args
 	var valueFiles value.StringOrStringList
 	var set value.StringOrStringList
 	var kubeVersion string
+	var skip_crds string
```
Comment (nicks) [inline internal/tiltfile/files.go:199]: shouldn't this be a boolean?

**Comment by nicks** [comment]
thanks! can you also add this to the API documentation? it's here:
https://github.com/tilt-dev/tilt/blob/master/internal/tiltfile/api/__init__.py#L724

**Comment by kahirokunn** [comment]
@nicks done. how about now?

File: internal/tiltfile/files.go (hunk 1 above)
Comment (kahirokunn) [reply internal/tiltfile/files.go]: done

**Review by nicks — APPROVED** [review]
LGTM!

**Comment by kahirokunn** [comment]
@nicks hello! May I ask when you plan to release it?

**Comment by kahirokunn** [comment]
Okay. Thx

**Pull request merged**